tests/php/Forms/TreeMultiselectFieldTest.php

```diff
@@ -10,22 +10,57 @@ class TreeMultiselectFieldTest extends SapphireTest
 {
     protected static $fixture_file = 'TreeDropdownFieldTest.yml';
 
+    public function testEmptyChoiceReadonly()
+    {
+        $field = new TreeMultiselectField('TestTree', 'Test tree', File::class);
+
+        $asdf = $this->objFromFixture(File::class, 'asdf');
+        $subfolderfile1 = $this->objFromFixture(File::class, 'subfolderfile1');
+
+        $schemaStateDefaults = $field->getSchemaStateDefaults();
+        $this->assertArraySubset(['id' => 'TestTree', 'name' => 'TestTree', 'value' => 'unchanged'], $schemaStateDefaults, $strict = true);
+
+        $items = $field->getItems();
+        $this->assertCount(0, $items, $message = 'there must be no items selected');
```

Comment:
`$strict` and `$message` appear to be unused

Reply:
oh, yeah, didn't notice the leftovers of autocompletion...